### ISSUE TITLE
Failed to apply patch to QDMA DPDK driver

### ISSUE THREAD

**Issue by zhiyisun** [Xilinx/open-nic-dpdk#4]
We may need to specify which commit (or branch, tag) we can apply the patch to the [QDMA DPDK](https://github.com/Xilinx/dma_ip_drivers) repo.

**Comment by function47** [comment]
From the timeline of two repos, this [commit](https://github.com/Xilinx/dma_ip_drivers/commit/785995783c78b2cbec6458141c4395e204c5bd9b) works for me.

**Comment by zhiyisun** [comment]
Yep, we can find the commit based on the time of this repo. QDMA DPDK driver updated after the last commit of this repo. We may need to add commit ID in the readme to tell the user which commit should be used.

**Comment by cneely-amd** [comment]
Thanks, I updated the README instructions towards checking out the commit #7859957 of the dma_ip_drivers based on the above suggestions.
